cristin-person/src/main/java/no/unit/nva/cristin/person/model/nva/Person.java

```diff
@@ -238,13 +237,16 @@ private Map<String, String> convertTypedValuesToMap(Set<TypedValue> typedValueSe
     /**
      * Converts NVA formatted employments to Cristin formatted employments.
      */
+    @SuppressWarnings("PMD.ReturnEmptyCollectionRatherThanNull")
     public static List<CristinPersonEmployment> mapEmploymentsToCristinEmployments(Set<Employment> employments) {
         if (isNull(employments)) {
             return null;
         }
-        return new ArrayList<>(employments).stream()
-                   .map(Employment::toCristinEmployment)
-                   .collect(Collectors.toList());
+        return
+            employments
```

Comment:
is `new Arraylist<>(employments)` redundant here? `toList `is maybe creating a brand new object with a copy of the data anyways?

Reply:
Java 17